src/loadsave.jl

```diff
@@ -218,5 +218,15 @@ function action(call::Symbol, libraries::Vector{ActionSource}, @nospecialize(fil
             push!(failures, (e, library, catch_backtrace()))
         end
     end
+    if call === :save # Try MimeWriter methods
+        try
+            return MimeWriter.save(file, args...; options...)
```

Comment:
You're overriding the registry to force MimeWriter. It's also listed as a valid format for PNGs: https://github.com/JuliaIO/FileIO.jl/blob/b779539b505ba5aea6634c5cb78b2d2cc6e49ac8/src/registry.jl#L146

So why doesn't that work?

Reply:
ignore everything I did please 😂